### PR TITLE
New package: zr-1.0.2

### PR DESCRIPTION
**Pull request by strec007**
#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture (amd64)

### CODE REVIEW

File: srcpkgs/zr/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'zr'
+pkgname=zr
+version=1.0.2
+revision=1
+build_style=cargo
+hostmakedepends="pkgconf"
```
Comment (classabbyamp) [inline srcpkgs/zr/template:6]: i think most packages use pkg-config not pkgconf. does it specifically need pkgconf?

File: srcpkgs/zr/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+makedepends="openssl-devel"
+short_desc="Quick, simple zsh plugin manager"
+maintainer="Petr Cizmar <xxb0322@xpc.ovh>"
+license="MPL-2.0"
+homepage="https://github.com/jedahan/zr"
```
Comment (classabbyamp) [inline srcpkgs/zr/template:11]:
```suggestion
homepage="https://github.com/jedahan/zr"
changelog="https://raw.githubusercontent.com/jedahan/zr/main/changelog.md"
```

**Comment by classabbyamp** [comment]
can you add the email you used to commit to your github account?

also, the body of the commit can be removed

**Comment by strec007** [comment]
All done.

**Pull request merged**